strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/main/java/org/carlspring/strongbox/nuget/file/AssemblyTargetFrameworkAdapter.java

```diff
@@ -0,0 +1,73 @@
+package org.carlspring.strongbox.nuget.file;
+
+import com.google.common.base.Joiner;
+import com.google.common.base.Strings;
+
+import java.text.MessageFormat;
+import java.util.EnumSet;
+import javax.xml.bind.annotation.adapters.XmlAdapter;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AssemblyTargetFrameworkAdapter
+        extends XmlAdapter<String, EnumSet<Framework>>
+{
+
+    private static final Logger LOG = LoggerFactory.getLogger(AssemblyTargetFrameworkAdapter.class);
```

Comment:
Cosmetic: Across our code base, the name of the loggers is always `logger`, (which I agree is against the typical convention for constant fields). Would you mind changing this, please?